SqlServer/Get-DefaultSqlServerName/Get-DefaultSqlServerName.psm1

```diff
@@ -15,23 +15,24 @@ function Get-DefaultSqlServerName
 
     Process
     {
-        $serverServices = (Get-WmiObject win32_Service -Computer $env:COMPUTERNAME | Where-Object { $PSItem.Name -match "MSSQL" -and $PSItem.PathName -match "sqlservr.exe" })
+        $serverServices = Get-CimInstance win32_Service |
+            Where-Object { $_.Name -match "MSSQL" -and $_.PathName -match "sqlservr.exe" }
         $servicePath = ""
 
         # No SQL Servers installed.
-        if ($serverServices -eq $null)
+        if ($null -eq $serverServices)
```

Comment:
Why the reverse order?

Reply:
Otherwise we get this warning:
> `$null` should be on the left side of equality comparisons.
> PSScriptAnalyzer([PSPossibleIncorrectComparisonWithNull](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/possibleincorrectcomparisonwithnull?view=ps-modules))

Reply:
Interesting.